CotEditor/Sources/Document+ScriptingSupport.swift

```diff
@@ -182,6 +182,18 @@ extension Document {
     }
     
     
+    /// whether replace tab with spaces
+    var autoExpandTab: NSNumber {
```

Comment:
It's just a matter of my personal preference, but I prefer to use `expandsTab` here for the property name instead of `autoExpandTab`.